exercises/12_hygienic_macros/README.md

```diff
@@ -4,20 +4,19 @@ To quote [the reference](https://doc.rust-lang.org/reference/macros-by-example.h
 
 > By default, all identifiers referred to in a macro are expanded as-is, and are
 > looked up at the macro's invocation site. This can lead to issues if a macro
-> refers to an item or macro which isn't in scope at the invocation site. To
+> refers to an item (i.e. function/struct/enum/etc.) or macro which isn't in scope at the invocation site. To
```

Comment:
nit: "scope at the invocation site. To" could be pushed to the next line etc.

Reply:
Thanks, will fix separately.